src/components/atoms/DatasetInfoBox.tsx

```diff
@@ -1,15 +1,45 @@
 import React from 'react';
-import { Pane, Heading, Text, Badge } from 'evergreen-ui';
+import { Pane, Heading, Text, Badge, Link, Strong } from 'evergreen-ui';
+import { format, formatDistanceToNow, isEqual } from 'date-fns';
 
 type DatasetInfoBoxProps = {
     title: string;
     description: string;
     tags: string[];
+    url: string;
+    updated: string;
+    published: string;
 };
 
-const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tags }) => {
+const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tags, url, updated, published }) => {
+    const updatedDate = new Date(updated);
+    const publishedDate = new Date(published);
+
+    const Published: React.FC = () => {
+        return (
+            <Text title={`Published: ${format(publishedDate, 'dd/MM/yyyy;HH:mm')}`} size={300} marginTop="0.15rem">
```

Comment:
Use [`formatISO`](https://date-fns.org/v2.21.1/docs/formatISO) which is a universal standard. That includes timezones and can be easily parsed back and interpreted.

Reply:
> Rekkefølgen år-måned-dag er norsk og internasjonal standard og mye brukt ved utveksling av data, i kronologiske lister m.m.

https://www.sprakradet.no/sprakhjelp/Skriveregler/Dato/#dato

Dette er en del av ISO-8601

Reply:
Okey doke